child_frame_id: global_cam
frame_id: map
qw: -0.49827501665075846
qx: 0.8662779062623305
qy: -0.03575320685095562
qz: 0.002510996477851385
x: 21.71913466156304
y: 2.351480753578569
z: 3.813577070116803
